modules/ROOT/pages/_partials/keycloak/implementing-authentication.adoc

```diff
@@ -98,7 +98,8 @@ Cordova::
 +
 --
 // end::excludeDownstream[]
-Authentication is enabled during initialization as described in the section xref:setting-up-sdk-{context}[]. To manually redirect to the login screen, use:
+Authentication is enabled during initialization as described in the section xref:setting-up-sdk-{context}[].
+To manually redirect to the login screen, add the following code to the `< ? >` file:
```

Comment:
@austincunningham  - what file should this be added to, do you know?

Reply:
Depends if we are talking about cordova the `./www/index.js` file , if we are talking about ionic then `./src/main.ts`

Reply:
we don't have info, revert? and handle ionic separately?

Reply:
It makes sense to me to handle ionic separately however, I think this requires a more detailed plan as to how Ionic is catered for in the docs. I think this is a Content Strategist piece of work and realistially should be incorporated into the content plan for GA imo.